backend/dune

```diff
@@ -0,0 +1,20 @@
+(library
```

Comment:
ça y est le code est en version 10.191 ?
Where is the God Emperor ?
Bring Your Own Spice !
Aussi .... c'est du scheme ça :D ?

Reply:
2. Pourquoi tout est indenté de 4 même le début du code :D ?
Si on utilisais le pattern ou ( et ) partagent soit leurs X soit leur Y ?

Reply:
c'est un fichier de config ceci, pas du scheme mais des s-expressions néanmoins
2. Rien n'est indenté de 4 ?
3. Le ( et ) qui partagent X/Y, NON c'est illégal

Reply:
s-expressions et scheme ya de l'overlap ?
3. c'est illégal où le
```
(blabla
....param
....param
)
```
?

Reply:
ps: sinon ça y est ça indente de UN içitte xD ?

Reply:
s-expression c'est le format générique "(a b c)", scheme est donc formé de s-expressions
3. partout ! 
4. oui l'indent de 1 c'est le format par défaut de dune, c'est ainsi, c'est pas un fichier qu'on va lire souvent hein !